mission_yaml/SEA55_M78.yml

```diff
@@ -0,0 +1,486 @@
+  angular_cmd:
+    source: AngCmd
+    long_name: glider angular command
+    standard_name: angular_command
+    units: degrees
+    coordinates: time depth latitude longitude
+  angular_pos:
+    source: AngPos
+    long_name: glider angular position
+    standard_name: angular_position
+    units: degrees
+    coordinates: time depth latitude longitude
+  voltage:
+    source: Voltage
+    long_name: glider voltage
+    standard_name: platform_voltage
+    units: V
+    coordinates: time depth latitude longitude
+  altimeter:
+    source: Altitude
+    long_name: glider altimeter reading
+    standard_name: glider_altimeter_reading
+    units: m
+    coordinates: time depth latitude longitude
+  dead_reckoning:
+    source: DeadReckoning
+    long_name: dead reckoning
+    standard_name: dead_reckoning
+    units: None
+    coordinates: time depth latitude longitude
+  nav_resource:
+    source: NAV_RESOURCE
+    long_name: nav resource
+    standard_name: nav_resource
+    units: None
+    coordinates: time depth latitude longitude
+  conductivity:
+    source: LEGATO_CONDUCTIVITY
+    long_name: water conductivity
+    standard_name: sea_water_electrical_conductivity
+    units: mS cm-1
+    coordinates: time depth latitude longitude
+    instrument: instrument_ctd
+    valid_min: '0.'
+    valid_max: '85.'
+    observation_type: measured
+    accuracy: '0.003'
+    resolution: '0.001'
+  temperature:
+    source: LEGATO_TEMPERATURE
+    long_name: water temperature
+    standard_name: sea_water_temperature
+    units: Celsius
+    coordinates: time depth latitude longitude
+    instrument: instrument_ctd
+    valid_min: '-5'
+    valid_max: '42'
+    observation_type: measured
+    accuracy: '0.002'
+    resolution: '0.00005'
+  pressure:
+    source: LEGATO_PRESSURE
+    long_name: water pressure
+    standard_name: sea_water_pressure
+    units: dbar
+    coordinates: time depth latitude longitude
+    valid_min: '0'
+    valid_max: '1000'
+    positive: down
+    reference_datum: sea-surface
+    instrument: instrument_ctd
+    observation_type: measured
+    accuracy: '0.5'
+    resolution: '0.01'
+    comment: ctd pressure sensor
+  salinity:
+    source: LEGATO_SALINITY
+    long_name: water salinity
+    standard_name: sea_water_salinity
+    units: PSU
+    coordinates: time depth latitude longitude
+    instrument: instrument_ctd
+    valid_min: '0.'
+    valid_max: '50.'
+    observation_type: calculated
+    accuracy: '0.02'
+    resolution: '0.01'
+  chlorophyll_tridente:
+    source: LEGATO_TRIDENTE_CHLOROPHYLL
+    long_name: chlorophyll
+    standard_name: concentration_of_chlorophyll_in_sea_water
+    units:  μg L-1
+    coordinates: time depth latitude longitude
+    instrument: instrument_flourometer
+    observation_type: calculated
+  phycocyanin_tridente:
+    source: LEGATO_TRIDENTE_PHYCOCYANIN
+    long_name: phycocyanin
+    standard_name: concentration_of_phycocyanin_in_sea_water
+    units: μg L-1
```

Comment:
we use SI units where possible, so  μg L-1 in SI will be mg m-3. Same for chlorophyull